src/style.scss

```diff
@@ -0,0 +1,143 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap");
+
+$--blue-color: #00acdc;
+$--gray-color: #616070;
+$--black-color: #060b35;
+$--white-color: #fff;
+$--standard-width: 166px;
+$--bold-font-weight: 700;
+$--medium-font-weight: 500;
+$--regular-font-weight: 400;
```

Comment:
all variables must be in sepatate file

Reply:
![image](https://github.com/mate-academy/layout_product-cards/assets/103724609/4ec911c4-7dac-4e27-8586-f37754d93452)

Reply:
+